streams/src/main/java/org/apache/kafka/streams/internals/metrics/ClientMetrics.java

```diff
@@ -58,6 +60,8 @@ private ClientMetrics() {}
     private static final String TOPOLOGY_DESCRIPTION_DESCRIPTION =
         "The description of the topology executed in the Kafka Streams client";
     private static final String STATE_DESCRIPTION = "The state of the Kafka Streams client";
+    private static final String ALIVE_STREAM_THREADS_DESCRIPTION = "The number of alive stream threads";
```

Comment:
Since StreamThreads have a number of states that people are familiar with watching, it might be worth being specific about what "alive" means. In particular, it seems like many will wonder if this just means "running", which it does not.